backend/game/player/player.py

```diff
@@ -2,13 +2,17 @@
 from game.values.locations import Locations
 
 
+# from game.cards.card import Card
+
+
 class Player:
     def __init__(self):
         self.maximum_health = self.current_health = constants.DEFAULT_HEALTH
         self.location = Locations.OUTSIDE
         self.is_alive = True
         self.victory_points = constants.DEATH_HIT_POINT
         self.energy = constants.DEFAULT_ENERGY_CUBE
+        self.card_on_hand = {}
```

Comment:
Dictionary assumes that there is going to be one of each card as the key, this is not true as the mimic card alone will allow mutiple copies of the same card to exist.  This data structure will need to be reworked, my suggestion would be a list